daml-lf/validation/src/test/scala/com/digitalasset/daml/lf/validation/StackSafeTyping.scala

```diff
@@ -185,7 +185,7 @@ class StackSafeTyping extends AnyFreeSpec with Matchers with TableDrivenProperty
       val ctx: Context = Context.None
       val env = Typing.Env(langVersion, pkgInterface, ctx)
       try {
-        val _: Type = env.typeOf(expr)
+        val _: Type = Typing.typeOf(env, expr)
```

Comment:
Use the new entry point

Reply:
Should we modify `runTest` so that it has the static type `Option[ValidationError]` rather than the generic type `T`?